Samples/MobileBuyIntegration/MobileBuyIntegration/ViewControllers/CartViewController.swift

```diff
@@ -189,6 +193,238 @@ class CartViewController: UIViewController, UITableViewDelegate, UITableViewData
+            containerView.topAnchor.constraint(equalTo: checkoutButton.topAnchor, constant: 16),
+            containerView.bottomAnchor.constraint(equalTo: checkoutButton.bottomAnchor, constant: -16),
+
+            // Checkout label constraints
+            checkoutLabel.leadingAnchor.constraint(equalTo: containerView.leadingAnchor),
+            checkoutLabel.centerYAnchor.constraint(equalTo: containerView.centerYAnchor),
+
+            // Total label constraints
+            totalLabel.trailingAnchor.constraint(equalTo: containerView.trailingAnchor),
+            totalLabel.centerYAnchor.constraint(equalTo: containerView.centerYAnchor),
+            totalLabel.leadingAnchor.constraint(greaterThanOrEqualTo: checkoutLabel.trailingAnchor, constant: 8)
+        ])
+
+        // Clear the button's default title since we're using custom labels
+        checkoutButton.setTitle("", for: .normal)
+    }
+
+    private func setupAcceleratedCheckoutButtons() {
+        guard let cartId = CartManager.shared.cart?.id else { return }
+
+        // Create accelerated checkout buttons
+        let acceleratedCheckoutButtonsView = AcceleratedCheckoutButtons(cartID: cartId.rawValue)
+            .wallets([.shopPay, .applePay])
+            .cornerRadius(10)
+            .onComplete { _ in
+                // Reset cart on successful checkout
+                CartManager.shared.resetCart()
+            }
+            .onFail { error in
+                print("Accelerated checkout failed: \(error)")
+            }
+            .onCancel {
+                print("Accelerated checkout cancelled")
+            }
+            .environment(appConfiguration.acceleratedCheckoutsStorefrontConfig)
+            .environment(appConfiguration.acceleratedCheckoutsApplePayConfig)
+
+        // Wrap in AnyView and create hosting controller
+        let acceleratedCheckoutsController = UIHostingController(rootView: AnyView(acceleratedCheckoutButtonsView))
+        acceleratedCheckoutsController.view.translatesAutoresizingMaskIntoConstraints = false
+        acceleratedCheckoutsController.view.backgroundColor = .clear
+
+        // Set height constraint for accelerated checkout buttons (2 buttons: Apple Pay + Shop Pay)
+        let heightConstraint = acceleratedCheckoutsController.view.heightAnchor.constraint(equalToConstant: 96)
+        heightConstraint.priority = UILayoutPriority(999) // High priority but not required
+        heightConstraint.isActive = true
+
+        // Add to button stack view (at index 0 to appear above checkout button)
+        addChild(acceleratedCheckoutsController)
+        buttonStackView.insertArrangedSubview(acceleratedCheckoutsController.view, at: 0)
+        acceleratedCheckoutsController.didMove(toParent: self)
+
+        acceleratedCheckoutHostingController = acceleratedCheckoutsController
+
+        // Update button container size after adding buttons
+        updateButtonContainerSize()
+    }
```

Comment:
This is the code to render the SwiftUI AcceleratedCheckout buttons in a UIKit ViewController